frontend/src/store/items.ts

```diff
@@ -19,12 +20,62 @@ const defaultState: ItemsState = {
   collectionById: {}
 };
 
-const state = ref<ItemsState>(defaultState);
+const state = ref<ItemsState>(cloneDeep(defaultState));
 
 /**
  * == CLASS CONSTRUCTOR ==
  */
 class ItemsStore {
+  /**
+   * == GETTERS ==
+   */
+  public getItemById = (id: string | undefined): BaseItemDto | undefined => {
+    return computed(() => {
+      if (id) {
+        return state.value.byId[id];
+      }
+    }).value;
+  };
```

Comment:
```suggestion
  public getItemById = (id: string | undefined): BaseItemDto | undefined =>
    computed(() => (id ? state.value.byId[id] : undefined)).value;
```

Wdyt of this? Same for others :)

Reply:
If reactivity works, I'm fine

Reply:
Ok so it seems it has tradeoffs: `This will use more memory, because each class instance will have its own copy of each function defined this way`. So we'll keep it this way for the getter, I'll just change the computed return value

Reply:
We only have one instance on the app though.